example/src/sections/Delete.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import { Button, SafeAreaView, StyleSheet, Text, View } from 'react-native';
+import {
+  HealthKit,
+  HealthKitDataType,
+  HealthKitUnitType,
+} from '@kilohealth/rn-fitness-tracker';
+import { HealthDataRecord } from '@kilohealth/rn-fitness-tracker/src/types';
+import { addDays, startOfDay } from 'date-fns';
+
+const data = {
+  key: HealthKitDataType.HeartRate,
+  unit: HealthKitUnitType.BeatsPerMinute,
+  amount: 60,
+  metadata: {
+    HKWasUserEntered: 1,
+    logId: 8290,
+  },
+};
+
+export const Delete = () => {
+  const [record, setRecord] = useState<HealthDataRecord | undefined>(undefined);
+
+  const getHeartRateUUIDText = () => {
+    const text = record === undefined ? 'undefined' : record.uuid;
```

Comment:
How about `record?.uuid ?? 'undefined' `

Reply:
For some reason this didn't work, it then returned blank to the string. But I will double check